Makefile

```diff
@@ -119,6 +119,10 @@ check:
 	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/check.sh --golangci-lint-config=./.golangci.yaml ./cmd/... ./pkg/... ./test/...
 	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/check-charts.sh ./charts
 
+.PHONY: check-docforge
+check-docforge:
+	@$(REPO_ROOT)/hack/check-docforge.sh
```

Comment:
Can't we just call the `@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/check-docforge.sh` instead of copying the script to each extension now?

Reply:
This is not the main script. The main script is this one: https://github.com/gardener/documentation/blob/master/.ci/check-manifest

The purpose of this script is just providing some configurations to the main script.
Currently the helper script is not so generic and we can not just reuse it.
Maybe if we make `repoPath`, `manifestPath`, `diffDirs` and the path of the docforge binary to be configured via script arguments then  we will be able to reuse the script from vendor/g/g.
The call will be something like this:
```
@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/check-docforgesh $(REPO_ROOT) $(REPO_ROOT)/.docforge/manifest.yaml ".docforge/;docs/" $(REPO_ROOT)/hack/tools/bin
```

Reply:
But if u want I can make the changes to the script in g/g and will make it more generic. This will work for repos which have vendored g/g but the other ones will still need their own script

Reply:
I think it makes sense to make the script in g/g generic enough such that at least the many extensions can use it. This will already greatly reduce the amount of duplication that would otherwise be required.

Reply:
Okay, will do this